prow/plugins/approve/approvers/owners.go

```diff
@@ -591,8 +593,9 @@ The full list of commands accepted by this bot can be found [here](https://go.k8
 Needs approval from an approver in each of these OWNERS Files:
 
 {{range .ap.GetFiles .org .project}}{{.}}{{end}}
-You can indicate your approval by writing `+"`/approve`"+` in a comment
-You can cancel your approval by writing `+"`/approve cancel`"+` in a comment
+OWNER approvers can indicate their approval by writing `+"`/approve`"+` in a comment
```

Comment:
Maybe `Approvers can...` ? The `OWNER approvers` phrase is a little confusing to me

Reply:
The problem is that "approve" means two differnet things in this context. :/

Reply:
How about `Approvers listed in OWNERS files...`?